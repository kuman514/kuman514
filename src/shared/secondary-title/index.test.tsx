import { render } from '@testing-library/react';

import SecondaryTitle from '.';

describe('SecondaryTitle', () => {
  it('should show as expected', () => {
    const { container } = render(
      <SecondaryTitle>
        If you win you are Maverick, if you lose you are Yasuo.
      </SecondaryTitle>
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should show as expected 2', () => {
    const { container } = render(
      <SecondaryTitle customClassName="text-black">
        Yasuo Maverick
      </SecondaryTitle>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
