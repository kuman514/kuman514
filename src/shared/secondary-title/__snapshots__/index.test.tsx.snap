// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryTitle should show as expected 1`] = `
<h2
  class="text-bold text-4xl undefined"
>
  If you win you are Maverick, if you lose you are Yasuo.
</h2>
`;

exports[`SecondaryTitle should show as expected 2 1`] = `
<h2
  class="text-bold text-4xl text-black"
>
  Yasuo Maverick
</h2>
`;
